eo-parser/src/main/antlr4/org/eolang/parser/Eo.g4

```diff
@@ -47,7 +47,7 @@ metas
 // Objects
 // Ends on the next line
 objects
-    : (object EOL?)* object
+    : (object EOL?)+
```

Comment:
@volodya-lombrozo let's add a syntax test (or typo test) that would catch how many `EOLs` can be placed at the end of the file

Reply:
@maxonfjvipon Is the test you suggest related to this PR?

Reply:
@maxonfjvipon you can add such a test to the `master` branch ahead of this PR, if you believe this is important

Reply:
@volodya-lombrozo when you change a grammar, I believe it's important to cover it with test

Reply:
@maxonfjvipon I added the test in the master, it doesn't pass :( This means that currently we allow multiple EOLs at the end of source file, see 3ff271c71